helm-charts/sep-service/templates/service.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.fullName }}-svc-{{ .Values.service.name }}
+  namespace: {{ .Values.namespace }}
+{{- if .Values.service.annotations }}
+  annotations:
+{{- range $key, $value := .Values.service.annotations }}
+      {{ $key }}: {{ $value }}
+{{- end }}
+{{- end }}
+  labels:
+    {{- if .Values.service.labels }}
+    {{- range $key, $value := .Values.service.labels }}
+    {{- end }}
+    {{- end }}
+    app.kubernetes.io/name: {{ .Values.fullName }}-svc-{{ .Values.service.name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  - protocol: TCP
+    port: {{ .Values.service.servicePort }}       # port number the service will listen on
+    targetPort: {{ .Values.service.targetPort }}  # port number pods listen on
+  selector:
+    app.kubernetes.io/name: {{ .Values.service.name }} 
```

Comment:
This might cause clashes with other deployments of the same chart. I think you need to use more labels to ensure you only match pods from a single deployment. `app.kubernetes.io/instance:` might be enough but it's worth checking